### DISCUSSION TITLE
Github.io pages not linking through properly

### DISCUSSION THREAD

**Discussion by MajicDesigns** [community/community#22720] [Other Feature Feedback, Questions, & Ideas]
<p>I create documentation using Doxygen from my code and it gets loaded into a docs directory of my repositories for online viewing. It all works ok <em>except</em> for one section where the files will not show (Error 404). The same html help files work fine on my local computer where they are generated.</p>
<p>An example is <a href="https://majicdesigns.github.io/MD_MIDIFile/files.html" rel="nofollow noopener">https://majicdesigns.github.io/MD_MIDIFile/files.html</a>. Clicking on any of the file names listed on this page results in an error finding the file.</p>
<p>Any ideas on what what could be wrong would be gratefully accepted as I have no idea where to start with this.</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/majicdesigns">@MajicDesigns</a>, welcome to the GitHub Support Community!</p>
<p>I believe the issue is that the file names begin with underscores. It’s actually a conflict with Jekyll. If you’re not using Jekyll to build your GitHub pages, adding a file named <strong>.nojekyll</strong> to the docs folder should resolve the issue.</p>
<p>More info:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/" target="_blank" rel="nofollow noopener" title="06:06AM - 30 December 2009">The GitHub Blog – 30 Dec 09</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2019/03/blog-card.png?fit=5001%2C2626" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/" target="_blank" rel="nofollow noopener">Bypassing Jekyll on GitHub Pages - The GitHub Blog</a></h3>

<p>It is now possible to completely bypass Jekyll processing on GitHub Pages by creating a file named .nojekyll in the root of your pages repo and pushing it to GitHub. This should only be necessary</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by MajicDesigns** [comment]
<p>Yes, that fixed the problem. Thanks.</p>
